src/pages/Artist/Artist.js

```diff
@@ -1,15 +1,34 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
-import { PageTitle } from '../../components'
 import { setPageTitle } from '../../utils/setPageTitle'
+import { useArtist } from '../../services/artists/query'
 
-export default function Artist() {
```

Comment:
Do you mean props instead of artistSlug ?

Reply:
you prefer (...props) ?
for me, we are not going to pass another values than the artist slug, so wathever, specify artistSlug is more typed for me

Reply:
in this case, you need to destructure your props object 
`export default function Artist({artistSlug})`